source/agora/cli/checkvtable/check.d

```diff
@@ -19,7 +19,10 @@ import std.format;
 import std.stdio;
 import std.string;
 
-version (Windows) { pragma(msg, "This does not support Windows."); } else:
+version (Windows)
+{
+    pragma(msg, "Checking virtual method offset : This does not support Windows.");
```

Comment:
How about inserting the module name instead of the text of "Checking virtual method offset"?

Reply:
To add to @linked0 's suggestion:
```suggestion
    pragma(msg, __MODULE__, " : This does not support Windows.");
```